src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -168,6 +168,47 @@ const PurchaseOrderListTable = ({
     Alert.error(translate('react.default.featureNotSupported', 'This feature is not currently supported'));
   };
 
+  const getStatusTooltip = (status) => {
+    switch (status) {
+      case 'PENDING':
+        return translate(
+          'react.purchaseOrder.status.pending.description.label',
+          'This Purchase Order is in progress and has not been finalized.',
+        );
+      case 'PLACED':
+        return translate(
+          'react.purchaseOrder.status.placed.description.label',
+          'This Purchase Order has been finalized and submitted.',
+        );
+      case 'PARTIALLY_RECEIVED':
+        return translate(
+          'react.purchaseOrder.status.receiving.description.label',
+          'Part of this order has been received. Other shipments with items from this order are yet to be received.',
+        );
+      case 'SHIPPED':
+        return translate(
+          'react.purchaseOrder.status.shipped.description.label',
+          'At least one item on this PO has been shipped out.',
+        );
+      case 'PARTIALLY_SHIPPED':
+        return translate(
+          'react.purchaseOrder.status.shipped.description.label',
+          'At least one item on this PO has been shipped out.',
+        );
+      case 'RECEIVED':
+        return translate(
+          'react.purchaseOrder.status.received.description.label',
+          'This order has been received by the receiver.',
+        );
+      case 'COMPLETED':
+        return translate(
+          'react.purchaseOrder.status.completed.description.label',
+          'This order has been shipped and received. Related purchase invoices have been posted.',
+        );
+      default:
+        return status;
+    }
+  };
```

Comment:
what about keeping it somehow on backend, for example having another property for `OrderSummaryStatus` and inside `def statusOptions` in `PurchaseApiController` add another property behind `variant` called e.g. `tooltip`?
Let @awalkowiak decide what to do with this, it is just my preposition.